src/runtime_src/core/tools/xbmgmt2/CMakeLists.txt

```diff
@@ -15,9 +15,9 @@ include_directories(${CMAKE_BINARY_DIR})
 
 file(GLOB XBMGMT_V2_BASE_FILES
   "xbmgmt.cpp"
-  "XBMgmtMain.cpp"
-  "XBUtilities.cpp"
-  "SubCmd.cpp"
+  "../common/XBMain.cpp"

```

Comment:
consistency please.   how about  #include "tools/common/" like you do in other files?